test/files/run/richs.scala

```diff
@@ -40,11 +43,96 @@ object RichCharTest1 extends RichTest {
 //     println(s1 == s2)
 //   }
 // }
+
+object RichByteTest extends RichTest {
+  override def test() = {
+    val sixteen = 16.toByte
+    println(sixteen.toBinaryString) // should be "10000"
+    println(sixteen.toHexString)    // should be "10"
+    println(sixteen.toOctalString)  // should be "20"
+    val max = 0x7F.toByte
+    println(max.toBinaryString) // should be "10000"
```

Comment:
The "should be" comments from here down seem to be off. Fixing or removing them is fine with me.

Reply:
if twitter hadn't disabled my account for no reason, I would have tweeted something like TFW lrytz calls you on your test code comments. I don't remember the context here, but I will attempt to re-engage. Also, I don't miss twitter or nytimes crossword. I kind of miss nytimes, which made me feel like I was a smart person. That's also the reason I pretend to understand scala, so I feel smarter than the java guy at work.

Reply:
Removed extraneous test code comments in case someone reads them.